packages/entrypoint/matbas.sh.tftpl

```diff
@@ -11,7 +11,7 @@ export __MATBAS_DOLLAR_ZERO="$${0}"
 MATBAS_HOME="$(dirname "$(dirname "$(realpath "$${BASH_SOURCE[0]}")")")"
 ENTRYPOINT="$${MATBAS_HOME}/dist/main.js"
 %{if matbas_build == "debug" ~}
-NODE_OPTIONS=(--trace-deprecation --enable-source-maps --require '@matanuska/telemetry')
+NODE_OPTIONS=(--trace-deprecation --enable-source-maps)
```

Comment:
We can add this require back in, since telemetry has been implemented for Node.js in `@matanuska/debug`.